wikimedia/wikipedia-ios
# Talk page voiceover improvements

**Phabricator:** https://phabricator.wikimedia.org/T318611

### Notes
* This PR adds voiceover support to Talk Pages. The goal is to provide better context to some elements on the screen and adjusts the focus when necessary.
* Feel free to suggest text changes that would provide better context to users

### Test Steps
1. With voiceover on, navigate the talk pages, try to post and reply to topics, and simulate errors to check the error feedback.
